Apps/Src/Temperature.c

```diff
@@ -420,22 +419,42 @@ int32_t Temperature_GetMaxTemperature(void) {
  * @brief Gives fan speed based on Average temperature of pack and past error values
  * @param InputTemp - current temperature
  * @param DesiredTemp - desired temperature
- * @return FanSpeed: 0-4000 PWM
+ * @return FanSpeed: 0-8
  */
 int32_t Temperature_PID_Output(int32_t InputTemp, int32_t DesiredTemp) {
```

Comment:
Why are you returning an int32 if the range of values is 0-8? Also, Idk what exactly your code does, and I'm kinda getting lost in the sauce with your algorithm (and the negative numbers). You should update the documentation for this

Reply:
fixed

Reply:
this isn't fixed